packages/ui/Survey/index.tsx

```diff
@@ -80,7 +80,7 @@ export const SurveyInline = ({
     onFileUpload,
     responseCount,
   ]);
-  return <div id={containerId} className="h-full w-full" />;
```

Comment:
Is there a need to remove this `h-full` class? It leads to a bug in questions without images, the date-question in the image below should be center aligned but is aligned with the top because of this, please add the class back.


<img width="464" alt="Screenshot 2023-12-14 at 5 01 53 PM" src="https://github.com/formbricks/formbricks/assets/54475686/45a30b02-cf88-4283-9479-bc65d3b40856">

Reply:
I removed the h-full class due to improper spacing below when incorporating a logo or image. 
![image](https://github.com/formbricks/formbricks/assets/66241082/7ff3bc98-5b85-4b98-a6c9-b37d940c9640)

The removal, however, led to unexpected issues in the user interface. so i will add it back.